classifiers/nlp/ZeroShotTFClassifier/__init__.py

```diff
@@ -69,7 +69,8 @@ def __init__(
 
     def predict(self,
                 data: "np.ndarray",
-                target_labels: "np.ndarray") -> "np.ndarray":
+                *args,
```

Comment:
if `target_labels` are needed they should be defined apart from `*args`, `**kwargs`. 

But anyway, right now if this is needed we may need to rethink the interface, since there is no Driver in `jina` that would be able to work with this.

Reply:
I think what should happen is that `target_labels` should be an attribute of the `instance` and not part of the `predict` interface. So it will be a `Classifier` for a closed set of `classes`